code/modules/projectiles/projectile/hookshot.dm

```diff
@@ -8,6 +8,10 @@
 	kill_count = 15
 	grillepasschance = 0
 	var/obj/effect/overlay/hookchain/last_link = null
+	var/failure_message = "With a CLANG noise, the chain mysteriously snaps and rewinds back into the hookshot."
```

Comment:
> CLANG
![crash course on clang](https://user-images.githubusercontent.com/110073/31524239-13c92552-af6d-11e7-993d-551891443c48.png)

DO NOT INVOKE HIS NAME

Reply:
With a CLANG noise, the server mysteriously snaps and rewinds back into the void